.github/workflows/actions.yml

```
@@ -10,7 +10,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@main
```

I think we should use `v1` for improving stability of the CI.

```suggestion
        uses: google-github-actions/setup-gcloud@v1
```

---

The example in official doc also use `v1`.

---

thank you for your proposal.
I fix the target path.